lib/tito.rb

```diff
@@ -0,0 +1,29 @@
+require "httparty"
+require_relative "tito/event"
+require_relative "tito/null_event"
+
+class Tito
+  include HTTParty
+
+  base_uri "https://api.tito.io/railsbridge-boston"
```

Comment:
There's some trickiness with loading environment variables and setting this that is causing a lot of conflicts right now. For the sake of getting this out soon, I'll punt this change for another time.